src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/Microsoft.NET.Sdk.WebAssembly.Pack.pkgproj

```diff
@@ -7,7 +7,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(RepoTasksDir)Microsoft.NET.Sdk.WebAssembly.Pack.Tasks\Microsoft.NET.Sdk.WebAssembly.Pack.Tasks.csproj" />
-    <ProjectReference Include="$(RepoRoot)src\mono\wasm\host\WasmAppHost.csproj" />
+    <ProjectReference Include="$(RepoRoot)src\mono\browser\host\WasmAppHost.csproj" />
```

Comment:
I wonder what was broken by this until this PR and what test would catch that if any.

Reply:
Presumably this will break .NET 9 Preview 1 workloads?

Reply:
got it now, this file was not moved and probably could stay put because it's shared.

Reply:
Yes, the apphost is shared